snapshot/lib/snapshot/test_command_generator_base.rb

```diff
@@ -88,7 +93,8 @@ def resolve_result_bundle_path(language, locale)
         language_key = locale || language
 
         unless Snapshot.cache[:result_bundle_path][language_key]
-          path = File.join(Snapshot.config[:output_directory], "test_output", language_key, Snapshot.config[:scheme]) + ".test_result"
+          ext = FastlaneCore::Helper.xcode_version.to_i >= 11 ? '.xcresult' : '.test_result'
```

Comment:
```suggestion
          ext = fastlanecore::Helper.xcode_at_least?(11) ? '.xcresult' : '.test_result'
```

Do we want to use this helper? ☝️

Reply:
Makes sense! 👌 Done! ✅